src/client/components/navigation-button/navigation-button.style.css

```
@@ -28,28 +28,23 @@ p {
   position: relative;
 }
 .item:focus-visible {
-  background-color: #ebecf0;
-  box-shadow: inset 2px 2px 8px #c9cbd8,
-    inset -3px -3px 10px rgba(255, 255, 255, 0.9);
+  background: var(--light-grey);
+  box-shadow: var(--debossed-shadow-medium);
 }
 .item:hover {
-  background-color: #ebecf0;
-  box-shadow: inset 2px 2px 8px #c9cbd8,
-    inset -3px -3px 10px rgba(255, 255, 255, 0.9);
-}
-.item:hover path {
-  color: #73abd7;
+  background: var(--light-grey);
+  box-shadow: var(--debossed-shadow-medium);
 }
+.item:hover path,
 .item:hover p {
-  color: #73abd7;
+  color: var(--light-blue);
 }
 .itemActive {
```

in general you should use kebab-case for class names
```suggestion
.item-active {
```